### ISSUE TITLE
Some Inflection data lost in the html file produced from kindle dictionary by KindleUnpack  

### ISSUE THREAD

**Issue by njpig** [kevinhendricks/KindleUnpack#36]
Hi,

I unpacked Oxford Dictionary of English kindle dictionary with KindleUnpack(latest version), found there was not any Inflection data in idx:infl for the word "registered".

However, when I looked up "registered" with this dictionary in my kindle,  the word "register" was matched.

There are more information for your reference: https://www.mobileread.com/forums/showthread.php?t=346226

**Comment by kevinhendricks** [comment]
Yes, dictionary support was never a focus for KindleUnpack.  So existing dictionary support is old and sketchy at best.  With limited documentation and no access to a kindle anymore, I doubt that is ever going to change. 

For example see Issue #11 for other dictionary related bugs.

That said, contributions to improving the dictionary support in KindleUnpack are always welcome.
